src/NzbDrone.Core/MediaFiles/MediaInfo/VideoFileInfoReader.cs

```diff
@@ -174,6 +176,20 @@ private VideoStream GetPrimaryVideoStream(IMediaAnalysis mediaAnalysis)
             return mediaAnalysis.VideoStreams.FirstOrDefault(s => !codecFilter.Contains(s.CodecName)) ?? mediaAnalysis.PrimaryVideoStream;
         }
 
+        private AudioStream GetPrimaryAudioStream(IMediaAnalysis mediaAnalysis)
```

Comment:
`GetPrimaryOrBestAudioStream` no?

Reply:
This is similar to getting the video stream, which isn't always the primary stream, I think the naming is fine to maintain that consistency.